umka/ui.um

```diff
@@ -790,6 +790,13 @@ fn (gui: ^Gui) textBox*(tb: ^TextBox, cfg: TextBoxConfig = {}) {
 		}
 
 		runes := utf8.decode(input.getStr())
+		for i:=0; i < len(runes); i++ {
```

Comment:
Can we filter these out in tophat itself instead?

Reply:
That's a good idea. I will implement that.